.evergreen/run-kerberos-tests.sh

```diff
@@ -24,8 +24,11 @@ set -o xtrace
 npm install kerberos@">=2.0.0-beta.0"
 npm run check:kerberos
 
-npm install kerberos@"^1.1.7"
-npm run check:kerberos
+if [ "$NODE_LTS_VERSION" != "latest" ] || [ $NODE_LTS_VERSION -lt 20 ]; then
```

Comment:
This will attempt to install 1.1.7 because `"20" != "latest"` - the kerberos tests then fail.

Reply:
Link to failures: https://evergreen.mongodb.com/task_log_raw/mongo_node_driver_next_rhel80_large_Node20_test_auth_kerberos_patch_ccf555d069f3bd0eb7e7b2cca0d5ecdc7d571332_645e43889ccd4e4552dfeca8_23_05_12_13_47_52/0?type=T#L189